Makefile

```diff
@@ -0,0 +1,69 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+DOCKCROSS_SCRIPT := 'dockcross.bash'
+XGBOOST_REPO := 'https://github.com/maropu/xgboost.git'
+XGBOOST_BRANCH := 'xgboost_v0.60_with_portable_binaries'
+HIVEMALL_HOME := "$(shell pwd)"
+HIVEMALL_OUT := "${HIVEMALL_HOME}/target"
+XGBOOST_OUT := "${HIVEMALL_OUT}/xgboost"
+HIVEMALL_LIB_DIR := "${HIVEMALL_HOME}/xgboost/src/main/resources/lib"
+CANDIDATES := 'linux-arm64' 'linux-armv6' 'linux-armv7' 'linux-ppc64le' 'linux-x64' 'linux-x86' 'windows-x64' 'windows-x86'
```

Comment:
supported platform of hivemall depends on Hadoop.

target os is linux, windows, mac osx.
main target platform is x86, x64.

mac osx support is only used for local testing though.
https://github.com/xerial/snappy-java/blob/master/Makefile#L167
https://github.com/multiarch/crossbuild